proofs/cbmc/poly_uniform_gamma1/Makefile

```
@@ -0,0 +1,54 @@
+# SPDX-License-Identifier: Apache-2.0
+
+include ../Makefile_params.common
+
+HARNESS_ENTRY = harness
+HARNESS_FILE = poly_uniform_gamma1_harness
+
+# This should be a unique identifier for this proof, and will appear on the
+# Litani dashboard. It can be human-readable and contain spaces if you wish.
+PROOF_UID = poly_uniform_gamma1
+
+DEFINES +=
+INCLUDES +=
+
+REMOVE_FUNCTION_BODY +=
+UNWINDSET +=
+
+PROOF_SOURCES += $(PROOFDIR)/$(HARNESS_FILE).c
+PROJECT_SOURCES += $(SRCDIR)/mldsa/poly.c $(SRCDIR)/mldsa/symmetric-shake.c $(SRCDIR)/mldsa/fips202/fips202.c
```

This CBMC proof shouldn't need the FIPS sources. 
```suggestion
PROJECT_SOURCES += $(SRCDIR)/mldsa/poly.c
```